R/sprinkle.R

```diff
@@ -1,4 +1,4 @@
-#' @name sprinkles
+#' @name sprinkle
```

Comment:
These five lines could be replaced by just `@export`:

* roxygen will automatically detect the name
* similarly, it will figure out what to export
* you're referring to those functions with `::` so you don't need to import them